igniter/bootstrap_repos.py

```diff
@@ -515,17 +503,27 @@ def get_versions_from_directory(
 
         """
         installed_version = OpenPypeVersion.get_installed_version()
-        if not compatible_with:
-            compatible_with = installed_version
-        _openpype_versions = []
+        openpype_versions = []
         if not openpype_dir.exists() and not openpype_dir.is_dir():
-            return _openpype_versions
+            return openpype_versions
 
         # iterate over directory in first level and find all that might
         # contain OpenPype.
         for item in openpype_dir.iterdir():
-
-            # if file, strip extension, in case of dir not.
+            # if the item is directory with major.minor version, dive deeper
+            try:
+                ver_dir = item.name.split(".")[
+                              0] == installed_version.major and \
+                          item.name.split(".")[
+                              1] == installed_version.minor  # noqa: E051
+                if item.is_dir() and ver_dir:
+                    _versions = OpenPypeVersion.get_versions_from_directory(
+                        item)
+                    if _versions:
+                        openpype_versions.append(_versions)
+            except IndexError:
+                pass
+            # if file exists, strip extension, in case of dir don't.
```

Comment:
Wouldn't the `item.name` match exactly with `f"{installed_version.major}.{installed_version.minor}"`?
So wouldn't this just be `if item.name == f"{installed_version.major}.{installed_version.minor}"` or alike?

```
major_minor = f"{installed_version.major}.{installed_version.minor}"
if item.is_dir() and item.name == major_minor:
    _versions = OpenPypeVersion.get_versions_from_directory(item)
    openpype_versions.extend(_versions)
```

And shouldn't we also "continue" if we've already identified it as a major minor subfolder, so add
```
    continue
```